test/cypress/integration/widgets/blueprint_num_spec.ts

```diff
@@ -0,0 +1,51 @@
+describe('Number of Blueprints widget', () => {
+    const widgetId = 'blueprintNum';
+
+    function clickOnWidget() {
+        cy.get('.statistic').click();
+    }
+
+    function verifyUrl(expectedPageId: string) {
+        cy.location('pathname').should('be.equal', `/console/page/${expectedPageId}`);
+    }
+
+    function setWidgetConfiguration(pageToOpenOnClick: string) {
+        cy.editWidgetConfiguration(widgetId, () => {
+            cy.setSingleDropdownValue('Page to open on click', pageToOpenOnClick);
+        });
+    }
+
+    describe('Tests without mockLogin', () => {
+        before(() => cy.activate().login().addWidget(widgetId));
+
+        it('Opens the default page on click', () => {
+            clickOnWidget();
+            verifyUrl('blueprints');
+        });
+    });
+
+    describe('Tests with mockLogin', () => {
```

Comment:
I think we don't need such description here as when concatenated the full name of those tests will be:
1. "Number of Blueprints widget Tests with mockLogin displays the correct number of resources"
2. "Number of Blueprints widget Tests with mockLogin opens chosen page after configuration change on click"

Having
```suggestion
    describe('', () => {
```
will make it more readable.